### DISCUSSION TITLE
Expose Express app to embed within another server (Remix)

### DISCUSSION THREAD

**Discussion by alexluong** [keystonejs/keystone#7239] [Feature requests]
Hi folks,

I've been following Keystone development for a while and really excited about the direction it's going. I'm also a big fan of Remix and would like to use the 2 together.

In my opinion, the cleanest way for the 2 to work together is by embedding Keystone app inside Remix Express server. In order to do so, users need to be able to create the Keystone app (admin UI and GraphQL API) instead of using Keystone CLI to start the server.

I also explored using `extendExpressApp` but the admin UI being at root path makes it impossible to go that route. Therefore, a potential alternative is enabling an option to prefix the admin UI route (with `/admin` for example). I think this may be a more viable option. I'm open to making a PR for this with a bit of help if you think this makes sense.

Let me know what you think! Thanks for your great work!

**Comment by emcdaniel-sungage** [comment]
Would be interested to hear thoughts on this from someone on the Keystone team.

**Comment by phifa** [comment]
How is this going? I would also like to use the two together.

**Comment by muco-rolle** [comment]
Was anyone able to implement this?
